### ISSUE TITLE
Does not output to saved .txt log file

### ISSUE THREAD

**Issue by GambetTV** [cs96and/FoundryVTT-damage-log#15]
When saving the chat log, which outputs to a .txt file, the only thing that exports is the timestamp and the token name. None of the HP information exports.

**Comment by cs96and** [comment]
30421624d46041588e5b8d80ef5847de46596402

**Comment by cs96and** [comment]
Fixed in 1.6.4.  Note that old damage log messages will still appear blank when exported to the text log.  New messages will contain some minimal information.  E.g...

```
---------------------------
[1/10/2022, 1:06:58 PM] Bob
HP: 10 -> 8 Temp: 5 -> 0
---------------------------
```
